addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/protocol/CommandTypeNotSupportedlException.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pioneeravr.protocol;
+
+/**
+ * Thrown when a command type is not supported by the channel
+ * 
+ * @author Antoine Besnard
+ */
+public class CommandTypeNotSupportedlException extends Exception {

```

Comment:
This is a typo, right? It should be CommandTypeNotSupportedException not CommandTypeNotSupportedlException.